Bottom: efc9bda2fb38b38318fa2a2d4c874aab147d47cb
Top:    e58eeeb9bd5751767fc5aa4f8d5e717a40522a44
Author: Amit Kucheria <amit.kucheria@linaro.org>
Date:   2016-09-13 11:51:45 +0530

From 8d3cbae17a49eb520105aaa8252e718fc0ef7a98 Mon Sep 17 00:00:00 2001
Subject: [PATCH 10/21] defconfig: 96b_carbon: Enable the SPI driver by default

Change-Id: If9880d32424e812a2ce68698a458da03a7acdc42
Signed-off-by: Amit Kucheria <amit.kucheria@linaro.org>
Signed-off-by: Ricardo Salveti <ricardo.salveti@linaro.org>


---

diff --git a/boards/arm/96b_carbon/96b_carbon_defconfig b/boards/arm/96b_carbon/96b_carbon_defconfig
index ae11c05c2..58155b7a4 100644
--- a/boards/arm/96b_carbon/96b_carbon_defconfig
+++ b/boards/arm/96b_carbon/96b_carbon_defconfig
@@ -17,6 +17,9 @@ CONFIG_UART_CONSOLE_ON_DEV_NAME="UART_1"
 # enable pinmux
 CONFIG_PINMUX=y
 
+# enable SPI
+CONFIG_SPI=y
+
 # enable GPIO ports A, B, C, D
 CONFIG_GPIO=y
 CONFIG_GPIO_STM32_PORTD=y
